modules/carto/src/sources/utils.ts

```diff
@@ -3,6 +3,7 @@ import {encodeParameter, MapType} from '../api/maps-api-common';
 import {DEFAULT_HEADERS, DEFAULT_PARAMETERS, MAX_GET_LENGTH} from './common';
 import {buildMapsUrlFromBase} from '../config';
 
+const REQUEST_CACHE = new Map();
```

Comment:
I'm a bit worried about this cache. For example, in Builder when we want to refresh data we make a new request with `Cache-Control: no-cache` and with this REQUEST_CACHE It wouldn't work right?

Reply:
I mean, we could refresh data multiple times with `Cache-Control: no-cache` so it always would be the same request

Reply:
Currently the way `CartoLayer` works [is the same](https://github.com/visgl/deck.gl/blob/master/modules/carto/src/layers/carto-layer.ts#L231-L241). If you have a `CartoLayer` and don't change any props it won't refetch the data. 

I think it is reasonable that the client doesn't issue another request for exactly the same URL&headers, if you need to force a refresh I would suggest that in Builder you change one of the headers, [If-Modified-Since](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#if-modified-since) seems appropriate in this case

Reply:
OK, in any case, it going to be a breaking change in Builder and we need to validate with infra the [If-Modified-Since](https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching#if-modified-since) approach